### ISSUE TITLE
Missing namespaces in forwarded message

### ISSUE THREAD

**Issue by oxoWrk** [horazont/aioxmpp#295]
When sending a message with a forwarded message, all containers in forwarded stanza  have no namespaces

**Comment by horazont** [comment]
@oxoWrk Thanks for your report.

Please show debug log output of the stanza, either from aioxmpp or from the server to which aioxmpp is connected (ideally both).

**Comment by oxoWrk** [comment]
I have a properly working implementation(Twisted + Wokkel). This is a message sent from this implementation:
```
<message xmlns="jabber:client" xml:lang="en" to="username@domain.com/jabber-0af9382c" from="devbot@dev.domain.com/17611380635000916509654274" type="chat" id="0841d029-e690-4d2a-a6ba-7b49da65e6fc">
  <body>&gt; Wednesday, June 5, 2019\n&gt; [11:08:45] username@domain.com:\n&gt; https://web.domain.com/devbot/patrick.jpg\n&gt; Hello Bot!\nHi man</body>
  <reference xmlns="urn:xmpp:reference:0" begin="0" type="forward" end="141">
    <forwarded xmlns="urn:xmpp:forward:0">
      <delay xmlns="urn:xmpp:delay" stamp="2019-06-05T11:08:45Z"/>
      <message to="devbot@dev.domain.com/17611380635000916509654274" id="9222f66a-5c7a-4d10-a505-73d72997ee3c" from="username@domain.com/jabber" type="chat">
        <body>https://web.domain.com/devbot/patrick.jpg Hello Bot!</body>
        <reference xmlns="urn:xmpp:reference:0" begin="0" type="media" end="40">
          <media>
            <file>
              <media-type>image/jpeg</media-type>
              <name>patrick.jpg</name>
              <height>195</height>
              <width>330</width>
              <size>7320</size>
              <desc>patrick</desc>
            </file>
            <uri>https://web.domain.com/devbot/patrick.jpg</uri>
          </media>
        </reference>
        <reference xmlns="urn:xmpp:reference:0" begin="42" type="markup" end="46">
          <bold/>
        </reference>
        <reference xmlns="urn:xmpp:reference:0" begin="48" type="mention" end="50">
          <uri>xmpp:devbot@dev.domain.com</uri>
        </reference>
      </message>
    </forwarded>
  </reference>
</message>
```

And this is the same message sent using aioxmpp

```
<message xmlns="jabber:client" xml:lang="en" to="username@domain.com/jabber-0af9382c" from="refbot@xmppdev01.domain.com/366731101244774912129908" type="chat" id="d46aa151-21c7-49ea-8893-a3923d9f1386">
  <body>&gt; Wednesday, June 5, 2019\n&gt; [11:08:45] username@domain.com:\n&gt; https://web.domain.com/devbot/patrick.jpg\n&gt; Hello Bot!\nHi man</body>
  <reference xmlns="urn:xmpp:reference:0" begin="0" end="141" type="forward">
    <forwarded xmlns="urn:xmpp:forward:0">
      <delay xmlns="urn:xmpp:delay" stamp="2019-06-05T11:08:45"/>
      <message from="username@domain.com/jabber" to="refbot@xmppdev01.domain.com/366731101244774912129908" id="25678896-583e-4d70-ac3e-a4b3b0acc003" type="chat">
        <body>https://web.domain.com/devbot/patrick.jpg\nHello Bot!</body>
        <reference begin="0" end="41" type="media">
          <media>
            <file>
              <media-type>image/jpeg</media-type>
              <name>patrick.jpg</name>
              <height>195</height>
              <width>330</width>
              <size>7320</size>
              <desc>patrick.jpg</desc>
            </file>
            <uri>https://web.domain.com/devbot/patrick.jpg</uri>
          </media>
        </reference>
        <reference begin="42" end="46" type="markup">
          <bold/>
        </reference>
        <reference begin="48" end="50" type="mention">
          <uri>xmpp:refbot@xmppdev01.domain.com</uri>
        </reference>
      </message>
    </forwarded>
  </reference>
</message>
```

And as you can see, namespace of the references  is missing in the forwarded message

**Comment by horazont** [comment]
Thanks. This looks like a bug indeed. Can you share example code which generates that to make debugging easier?

**Comment by horazont** [comment]
@oxoWrk I was able to reproduce the issue, see #296. I’m on it.

**Comment by horazont** [comment]
@oxoWrk If you can test patches, it would be great if you could test [this fix])https://patch-diff.githubusercontent.com/raw/horazont/aioxmpp/pull/296.patch).

**Comment by oxoWrk** [comment]
I tested fix and and now it looks great. 
Thank you

**Comment by horazont** [comment]
@oxoWrk Thanks for testing! We’ll proceed with merging the fix into the 0.11 branch and make a bugfix release for 0.10.
